starboard/build/config/BUILD.gn

```diff
@@ -14,6 +14,10 @@
 
 import("//build/config/compiler/compiler.gni")
 
+config("hacky_build_config") {
+  defines = [ "USE_HACKY_COBALT_CHANGES" ]
```

Comment:
What's the purpose of introducing this USE_HACKY_COBALT_CHANGES? If we have to add it, maybe better add more comments to provide some context.